apps/console/src/features/notification-channels/pages/notification-channels.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Grid from "@oxygen-ui/react/Grid";
+import { TestableComponentInterface } from "@wso2is/core/models";
+import { PageLayout } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement } from "react";
+import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { Divider } from "semantic-ui-react";
+import emailIcon from "../../../themes/default/assets/images/icons/email-icon.svg";
+import smsIcon from "../../../themes/default/assets/images/icons/sms-icon.svg";
+import { AppConstants, AppState, FeatureConfigInterface, history } from "../../core";
+import { SettingsSection } from "../settings/settings-section";
+
+/**
+ * Props for the Server Configurations page.
+ */
+type NotificationChannelPageInterface = TestableComponentInterface;
```

Comment:
```suggestion
type NotificationChannelPageInterface = IdentifiableComponentInterface;
```
Let's use `data-componentid` on new UIs instead of `data-testid`